coderd/healthcheck/database.go

```diff
@@ -10,20 +10,30 @@ import (
 	"github.com/coder/coder/v2/coderd/database"
 )
 
+const (
+	DatabaseDefaultThreshold = 15 * time.Millisecond
+)
+
 // @typescript-generate DatabaseReport
 type DatabaseReport struct {
-	Healthy   bool    `json:"healthy"`
-	Reachable bool    `json:"reachable"`
-	Latency   string  `json:"latency"`
-	LatencyMs int     `json:"latency_ms"`
-	Error     *string `json:"error"`
+	Healthy     bool    `json:"healthy"`
+	Reachable   bool    `json:"reachable"`
+	Latency     string  `json:"latency"`
+	LatencyMs   int64   `json:"latency_ms"`
```

Comment:
Do we typically style these as Ms or MS? I would think it's MS but I didn't verify, I don't really care which but would prefer consistency. Alt type it out, i.e. Millis or Milliseconds, but again, whichever is more consistent.